### ISSUE TITLE
Multiple index entries for packages that differ only in case

### ISSUE THREAD

**Issue by neilb** [andk/pause#355]
PAUSE was fixed so that indexing permissions are now handled case-insensitively, and we auto switch to the latest version. So if you release module **frobscottle** and in a later release change it to **Frobscottle**, you'll end up with first-come on Frobscottle.

And we resolved all the historical cases, so for example CCOLBOURN only has first-come on `AI::NNFlex::Backprop`, even though he first released `AI::NNFlex::backprop`.

But we're still ending up with multiple entries in the index for case-differing packages:

    AI::NNFlex::backprop    undef    C/CC/CCOLBOURN/ai-nnflex/AI-NNFlex-0.20.tar.gz
    AI::NNFlex::Backprop    undef    C/CC/CCOLBOURN/ai-nnflex/AI-NNFlex-0.24.tar.gz

Depending on how CPAN clients process the index, and handle case-insensitivity of package names, this could result in someone installing the wrong version of AI-NNFlex (the old version appears in the index ahead of the new version).

I haven't checked to see whether this only applies to historical releases still on CPAN, or whether this can happen today. There are other cases -- AI-NNFlex was just the first I came across.

**Update**: have confirmed that if you do this now, you only end up with the more recent version being listed in the index, so it's just the historical cases that are in the index.

**Comment by wolfsage** [comment]
How was the historical data cleaned up? It appears that all but *1* of the mismatched case issues are before the fix was merged in 2013.

The other case I cannot reproduce but was two dists uploaded by perlancar in quick succession I think...
```
Data-Sah-Coerce-perl-To_duration-From_str-Alami_EN-0.006.tar.gz
Modify: 2019-11-28 18:51:58.000000000 -0500
Data-Sah-Coerce-perl-To_duration-From_str-alami_en-0.007.tar.gz
Modify: 2019-11-28 18:52:09.000000000 -0500
```

So it seems like the historical data wasn't cleaned up, or if it was... somehow old dists were reindexed??

**Comment by neilb** [comment]
Over the last few weeks I've been working through the duplicates. For many of them I managed to get in touch with the author, and got their permission to delete old releases. There were a few, mainly very old, cases where I couldn't track the author down, so I went through those with Andreas, and he manually deleted them from the index.

So as of last night, 21st May 2023, there are none of these left in the DB.
